src/Symfony/Bridge/Twig/DataCollector/TwigDataCollector.php

```diff
@@ -31,7 +31,7 @@ class TwigDataCollector extends DataCollector implements LateDataCollectorInterf
     private $twig;
     private $computed;
 
-    public function __construct(Profile $profile, Environment $twig)
+    public function __construct(Profile $profile, Environment $twig = null)
```

Comment:
Maybe passing `null` should trigger a deprecation warning, so we can remove the boilerplate below on the 4.0 branch again.

Reply:
sounds reasonable :) lets see what others think.

Reply:
No need to "deprecate" that as we are talking about an additional feature.